sql/synth_recipes.sql

```diff
@@ -2075,7 +2075,7 @@ INSERT INTO `synth_recipes` VALUES (29945,1,0,0,0,49,0,0,0,0,0,4100,4242,13979,0
 INSERT INTO `synth_recipes` VALUES (29947,1,0,0,0,12,0,0,0,0,0,4100,4242,12705,0,0,0,0,0,0,0,650,850,649,649,1,1,1,1,'Brass Mittens (desynth)'); -- 2022.03.04 Base on FFXIclopedia Strange result, normal result would have bronze ingot yield and brass ingot hq3
 INSERT INTO `synth_recipes` VALUES (29948,1,0,0,0,71,0,0,0,0,0,4100,4242,12802,0,0,0,0,0,0,0,914,851,931,745,1,1,1,1,'Gold Cuisses (desynth)'); -- 2022.03.04 Base on FFXIclopedia
 INSERT INTO `synth_recipes` VALUES (29949,1,0,0,0,16,0,0,0,0,0,4100,4242,12833,0,0,0,0,0,0,0,817,850,649,650,3,1,1,1,'Brass Subligar (desynth)'); -- 2022.03.04 Base on FFXIclopedia
--- INSERT INTO `synth_recipes` VALUES (29950,1,0,0,0,30,0,0,0,0,0,4100,4242,16451,0,0,0,0,0,0,0,1226,1226,1226,1226,6,6,6,6,'Mythril Dagger (desynth)'); -- 2022.03.04 Base on FFXIclopedia normal yield abnormal and no hq result
+INSERT INTO `synth_recipes` VALUES (29950,1,0,0,0,30,0,0,0,0,0,4100,4242,16451,0,0,0,0,0,0,0,1226,1226,1226,1226,6,6,6,6,'Mythril Dagger (desynth)'); -- 2022.03.04 Base on FFXIclopedia normal yield abnormal and no hq result
```

Comment:
incorrect rewards from desynthesis.

Reply:
> incorrect rewards from desynthesis.

Yeah I have not been able to find any other source and my retail char does not have smithing. I will remove that part of PR until can get some kind of verification.

Reply:
@TracentEden https://ffxi.allakhazam.com/db/item.html?fitem=48#Desynthesis